### PR TITLE
Cu 9evhd2 confirmation box width

### PR DESCRIPTION
**Pull request by Hamza-A**
## Description
This PR aims to resize the `<ConfirmationBox />` so that it doesn't stretch accross the width of the page on the desktop version

- I isolated `PrimaryConfirmationBox` and `DeleteConfirmationBox` in apart functions for more readability in `<IdentityOverview />`
- I resized `<ConfirmationBox />` on desktop version giving it `width: 52.8rem` (because `<Main />`'s `width: 88rem` and the `<ChildrenContainer />`'s `width: 60%`, hence 88 * 0.6 = 52.8
- I changed the `left` property of `<Box />` styled component in `<ConfirmationBox />` to 41% so that the box would appear aproximately below the `<ChildrenContainer />`
- I added an `<a />` tag into `<Link />` where needed
## Type of change

- [x] **Chore** (non-breaking change which refactors / improves the existing code base).
- [x] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## How Has This Been Tested?
There were no needs to add or modify tests

## Screenshots
image

## Checklist:

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

File: src/components/display/fewlines/IdentityOverview/IdentityOverview.tsx [hunk 1]
```diff
@@ -15,6 +15,90 @@ type IdentityOverviewProps = {
   identity: Identity;
 };
 
+const PrimaryConfirmationBox = (
```
Comment (Fuzznimp) [inline src/components/display/fewlines/IdentityOverview/IdentityOverview.tsx:18]: Can you move these two components into a dedicated file?

File: src/components/display/fewlines/ConfirmationBox/ConfirmationBox.tsx [hunk 2]
```diff
@@ -1,19 +1,40 @@
...
   display: flex;
   flex-direction: column;
   align-items: center;
   padding: 4rem 2rem 3rem;
   position: fixed;
-  left: 0;
+  left: 41%;
   background-color: ${({ theme }) => theme.colors.background};
-  width: 100%;
+  width: 52.8rem;
```
Comment (Fuzznimp) [inline src/components/display/fewlines/ConfirmationBox/ConfirmationBox.tsx:32]: Could you try `width: auto` instead?

File: src/components/display/fewlines/IdentityOverview/IdentityOverview.tsx (hunk 1 above)
Comment (Hamza-A) [reply src/components/display/fewlines/IdentityOverview/IdentityOverview.tsx]: OK done 65e8036

File: src/components/display/fewlines/ConfirmationBox/ConfirmationBox.tsx (hunk 2 above)
Comment (Hamza-A) [reply src/components/display/fewlines/ConfirmationBox/ConfirmationBox.tsx:32]: The problem with `width: auto;` is that the box's size will vary according to the content, so we will have different sizes for `<PrimaryConfirmationBox />` and `<DeleteConfirmationBox />`

Comment (Fuzznimp) [reply src/components/display/fewlines/ConfirmationBox/ConfirmationBox.tsx:32]: Well, is that an issue?

Comment (Hamza-A) [reply src/components/display/fewlines/ConfirmationBox/ConfirmationBox.tsx:32]: Not really actually, I was just following the wireframe. A fixed `width` would help me align the Box as much as possible and whatever the content though.

With `width: auto;` it seems that the `<DeleteConfirmationBox />` is out of line (even though without the Bow wasn't perfectly aligned).

image

What is your opinion?

Comment (Fuzznimp) [reply src/components/display/fewlines/ConfirmationBox/ConfirmationBox.tsx:32]: Ok,we'll refacto everything later then.

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**